charts/dragonfly-operator/templates/crds.yaml

```diff
@@ -0,0 +1,1611 @@
+{{- if .Values.crds.install }}
```

Comment:
There's a long lengthy debate on how CRDs should be managed in Helm charts. Either by what you did here, or dropping them un-templated in a `crds` folder and letting helm install them by default. If your team is okay with this method then it shouldn't be a problem moving forward.

I honestly prefer the `crds` folder method they describe [here](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you) but there are some caveats the main one being the CRDs will not be upgraded even if you change them here in the chart. There's ways around that by using Flux, Argo or any other CI/CD tool.